src/sourabh/Assignment_12/Test.java

```diff
@@ -0,0 +1,43 @@
+package sourabh.Assignment_12;
+
+public class Test {
+	
+	double sum(double firstNumber, double secondNumber, double thirdNumber) {
+		double totalNumber=firstNumber+secondNumber+thirdNumber;
+		return totalNumber;
+	}
+	double average(double firstNumber, double secondNumber, double thirdNumber) {
+		Test test= new Test();
+		double averageAns= test.sum(firstNumber, secondNumber, thirdNumber)/3;
+		System.out.println("The Average is : "+averageAns);
+		return averageAns;
+	}
+	boolean isAnswerAboveExpecation(double averageAns){
+		if(averageAns>50)
+			return true;
+		else
+			return false;
+	}
+	static String isEligible(double averageAns) {
+		String candidate;
+		if(averageAns>50 || averageAns %2==0) {
```

Comment:
we need to use divisible function not modular.
if value is more than 50 or divisible by 2. Eligibility will be printed in main method.